docs/application-tutorial/game_of_life.md

```diff
@@ -0,0 +1,48 @@
+# Game of Life
```

Comment:
Maybe we can put a gif of the script running, wdyt?

Reply:
ah yes, indeed, let me do that

Reply:
done